src/command/ping-command.ts

```diff
@@ -125,6 +146,21 @@ export class PingCommand implements CommandInterface<PingOptions> {
 		return true;
 	}
 
+	private toJsonOutput(input: PingParseOutput): PingParseOutputJson {
+		return {
+			rawOutput: input.rawOutput,
+			resolvedAddress: input.resolvedAddress ? input.resolvedAddress : null,
+			resolvedHostname: input.resolvedHostname ? input.resolvedHostname : null,
+			timings: input.timings ?? [],
+			stats: {
+				min: input.stats?.min ?? null,
```

Comment:
Looks like we're inconsistent with the timings. Here all default to null, but for HTTP, they remain undefined?

Reply:
In this case, timings is an empty array. Stats are defaulted to null.

HTTP timings are internally inconsistent, it makes more sense to not render them at all if theyre missing; `tls` on HTTP request, and HTTP2 doesn't report `tls`, `tcp` or `dns` value.

Reply:
I've set `total` and `download` to 0 by default, and anything else goes to null.